patches/components-download-public-background_service-clients.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/download/public/background_service/clients.h b/components/download/public/background_service/clients.h
+index 6f0575113bade18b5daca40c227d293c2caddcef..49fdbe93214d90f8ff47741ea2cf5faa8dafbe27 100644
+--- a/components/download/public/background_service/clients.h
++++ b/components/download/public/background_service/clients.h
+@@ -45,6 +45,8 @@ enum class DownloadClient {
+ 
+   // New clients should be added above here.
+   BOUNDARY = 7,
++
++  CUSTOM_LIST_SUBSCRIPTIONS = 256,
```

Comment:
@bridiver what's the right value/behavior to choose here? Just the next free value after `BOUNDARY`? Bump `BOUNDARY` and replace it?